code/modules/admin/topic.dm

```diff
@@ -1467,6 +1483,8 @@
 		if(istype(M, /mob/new_player))
 			to_chat(usr, "This cannot be used on instances of type /mob/new_player")
 			return
+		if(alert(usr, "Confirm?", "Yes", "No") != "Yes")
+			return

```

Comment:
Instead of putting a confirmation here, you can change the two Animalize() procs in transform_procs.dm so their input boxes accept null (Cancel), and abort if null. No need for an extra confirmation box.